src/Unosquare.Labs.EmbedIO/Modules/StaticFilesModule.cs

```diff
@@ -185,6 +186,7 @@ public string DefaultExtension
         /// <value>
         ///   <c>true</c> if [use ram cache]; otherwise, <c>false</c>.
         /// </value>
+        [Obsolete("Explicit RamCache will be dropped in future versions")]
```

Comment:
RamCache won't be dropped but replaced by FileCachingMode (read-only).

Reply:
Should I change the message?

Reply:
Maybe "Caching options will be read-only in future versions." on the setter,
and "RAMCache will be replaced by similar functionality." (or even nothing) on the property.